tests/FakeItEasy.Specs/ArgumentEqualityComparerSpecs.cs

```diff
@@ -93,6 +147,16 @@ public class CustomComparer : ArgumentEqualityComparer<ClassWithCustomArgumentEq
             }
         }
 
+        public class CustomComparerThatIsNotNullSafe : ArgumentEqualityComparer<ClassWithCustomArgumentEqualityComparerThatIsNotNullSafe>
+        {
+            protected override bool AreEqual(
+                ClassWithCustomArgumentEqualityComparerThatIsNotNullSafe? expectedValue,

```

Comment:
Doesn't this produce a nullability warning? The argument is not nullable in the base class.

Reply:
I didn't notice, but I do have some nullability changes coming.